SIGSRE-65: Add SLO Bootstrap Guide

lisa opened the pull request:
There is still a link to be filled in for the SLO Reporting content, but
it's a work in progress still. The link now points to a Google Doc
(which is not likely to be available to folks outside Red Hat) where
ideation is being fleshed out.

Signed-off-by: Lisa Seelye <lisa@users.noreply.github.com>

lisa committed:
SIGSRE-65: Add SLO Bootstrap Guide

There is still a link to be filled in for the SLO Reporting content, but
it's a work in progress still. The link now points to a Google Doc
(which is not likely to be available to folks outside Red Hat) where
ideation is being fleshed out.

Signed-off-by: Lisa Seelye <lisa@users.noreply.github.com>

jeremyeder commented on slo_bootstrap_guide.md:
```diff
@@ -0,0 +1,46 @@
+# SLO Bootstrap Guide
+
+Red Hat's SIG-SRE group has created a set of documents which share how some teams engage with the practice of SLOs inside Red Hat. It is our hope that these supporting documents can assist others. Before we get started, there are some [definitions](#step-1-familiarise-definitions) to ensure common understanding of terms and phrases that are used throughout the package.
```
Add a sentence here "This guide is meant to be a self-contained bootstrapping guide for teams looking to use the site reliability engineering approach to supporting services."

lisa committed:
SIGSRE-65: PR Feedback

Add a note from Jeremy about this doc being a self-contained bootsrappign guide.

Signed-off-by: Lisa Seelye <lisa@users.noreply.github.com>

mh21 reviewed (approved):
LGTM!
